sphere-models/src/main/java/io/sphere/sdk/orders/expansion/OrderExpansionModel.java

```diff
@@ -11,20 +10,21 @@
 
  @param <T> the type for which the expansion path is
  */
-public class OrderExpansionModel<T> extends ExpansionModel<T> {
+public class OrderExpansionModel<T> extends CartLikeExpansionModel<T> {
     public OrderExpansionModel(final Optional<String> parentPath, final String path) {
-        super(parentPath, Optional.of(path));
+        super(parentPath, path);
     }
 
     OrderExpansionModel() {
         super();
     }
 
-    public static OrderExpansionModel<Order> of() {
-        return new OrderExpansionModel<>();
+    public SyncInfoExpansionModel<T> syncInfo() {
+        //since it is a set, there is no method with index access
+        return new SyncInfoExpansionModel<>(pathExpressionOption(), "syncInfo[*]");
     }
 
-    public ExpansionPath<T> customerGroup() {
```

Comment:
It is in the base class CartLikeExpansionModel, I wan't share the expansions of cart/order so I don't need to write too much tests.